presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -231,16 +247,29 @@ private MaterializedResult getExpectedOrdersTableDescription(boolean dateSupport
         }
         else {
             return MaterializedResult.resultBuilder(getQueryRunner().getDefaultSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
-                    .row("orderkey", "bigint", "", "")
-                    .row("custkey", "bigint", "", "")
-                    .row("orderstatus", "varchar", "", "")
-                    .row("totalprice", "double", "", "")
-                    .row("orderdate", orderDateType, "", "")
-                    .row("orderpriority", "varchar", "", "")
-                    .row("clerk", "varchar", "", "")
-                    .row("shippriority", "integer", "", "")
-                    .row("comment", "varchar", "", "")
-                    .build();
+                .row("orderkey", "bigint", "", "")
+                .row("custkey", "bigint", "", "")
+                .row("orderstatus", "varchar", "", "")
+                .row("totalprice", "double", "", "")
+                .row("orderdate", orderDateType, "", "")
+                .row("orderpriority", "varchar", "", "")
+                .row("clerk", "varchar", "", "")
+                .row("shippriority", "integer", "", "")
+                .row("comment", "varchar", "", "")
+                .build();
+        }
+    }
+
+    @Test
+    public void testCreateSchema()
+            throws SQLException
+    {
+        skipTestUnless(canCreateSchema());
+        assertUpdate("CREATE SCHEMA test_create_schema");
+        assertQuery("SHOW SCHEMAS LIKE 'test_create_schema'", "VALUES 'test_create_schema'");
+        cleanUpSchema();
+        if (canDropSchema()) {
+            assertUpdate("DROP SCHEMA test_create_schema");
```

Comment:
add dedicated test for drop which is not run when connector does not support that

Reply:
schema is already dropped here